samza-core/src/main/scala/org/apache/samza/checkpoint/CheckpointTool.scala

```diff
@@ -152,25 +153,34 @@ object CheckpointTool {
     val cmdline = new CheckpointToolCommandLine
     val options = cmdline.parser.parse(args: _*)
     val config = cmdline.loadConfig(options)
-    val rconfig = rewriteConfig(new JobConfig(config))
-    print("Rewritten config" + rconfig)
-    val tool = CheckpointTool(rconfig, cmdline.newOffsets)
-    tool.run
+    val rewrittenConfig = rewriteConfig(new JobConfig(config))
+    info(s"Using the rewritten config: $rewrittenConfig")
+    val tool = CheckpointTool(rewrittenConfig, cmdline.newOffsets)
+    tool.run()
   }
 }
 
-class CheckpointTool(config: Config, newOffsets: TaskNameToCheckpointMap, manager: CheckpointManager) extends Logging {
+class CheckpointTool(newOffsets: TaskNameToCheckpointMap, coordinatorStreamManager: CoordinatorStreamManager) extends Logging {
+
+  def run() {
+    // Read the configuration stored in the coordinator stream.
+    coordinatorStreamManager.register(getClass.getSimpleName)
+    coordinatorStreamManager.start()
+    coordinatorStreamManager.bootstrap()
+    val coordinatorStreamConfig: Config = coordinatorStreamManager.getConfig
```

Comment:
`coordinatorStreamConfig` seems to be used elsewhere to indicate the Config about a coordinator stream which is used to build a coordinator stream consumer/manager. This here seems to be the full job config loaded from the coordinator stream, so maybe use `configFromCoordinatorStream` to clarify that?

Reply:
Sure. Applied the suggestion.